tests/composefiles/aci_secrets/compose-invalid-target.yml

```diff
@@ -1,16 +0,0 @@
-services:
```

Comment:
I think it would be good to have several services using different secrets of the same volume, to ensure each one has only access to the right secrets

Reply:
This file was removed. But I can add several services in the unit test that replaces this one